api/authlete_api_impl.go

```diff
@@ -42,7 +42,12 @@ type impl struct {
 }
 
 func (self *impl) init(configuration conf.AuthleteConfiguration) {
-	self.baseUrl = configuration.GetBaseUrl()
+	// validate base URL
```

Comment:
Looks good! Thank you very much.
One thing - we probably should remove this comment to keep things tidy.

Reply:
OK! removed.
https://github.com/authlete/authlete-go/pull/13/commits/f2ea2dffd33392211fdf4ea7775a6bcbde005eaf